lib/kube/proxy/forwarder.go

```diff
@@ -2770,3 +2776,25 @@ func errorToKubeStatusReason(err error, code int) metav1.StatusReason {
 		return metav1.StatusReasonUnknown
 	}
 }
+
+// formatForwardResponseError formats the error response from the connection
+// monitor to a Kubernetes API error response.
+type formatForwardResponseError func(status *apierrors.StatusError) error
+
+func (f formatForwardResponseError) WriteString(s string) (n int, err error) {
+	if f == nil {
+		return len(s), nil
```

Comment:
```suggestion
		return 0, nil
```

Since nothing was written, correct? Or would this cause problems?

Should we error instead?

Reply:
It's a no-op so I think we should return len(s) instead of zero in case something is checking the value of `n` to adjust the buffer